cmd/record/main.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/joho/godotenv"
+	"github.com/kelseyhightower/envconfig"
+
+	"github.com/skygeario/skygear-server/pkg/core/auth/authn/resolver"
+	"github.com/skygeario/skygear-server/pkg/core/config"
+	"github.com/skygeario/skygear-server/pkg/core/logging"
+	"github.com/skygeario/skygear-server/pkg/core/middleware"
+	"github.com/skygeario/skygear-server/pkg/core/server"
+	"github.com/skygeario/skygear-server/pkg/record"
+	"github.com/skygeario/skygear-server/pkg/record/handler"
+)
+
+type configuration struct {
+	DevMode bool `envconfig:"DEV_MODE"`
+}
+
+func main() {
+	envErr := godotenv.Load()
+	if envErr != nil {
+		log.Print("Error in loading .env file")
+	}
+
+	configuration := configuration{}
+	envconfig.Process("", &configuration)
+
+	// logging initialization
+	logging.SetModule("record")
+
+	recordDependency := record.NewDependencyMap()
+
+	authContextResolverFactory := resolver.AuthContextResolverFactory{}
+	srv := server.NewServer("localhost:3000", authContextResolverFactory)
```

Comment:
Record gear is using same port as the auth gear. It is ok for deployment as they are running in different container. But may be inconvenient for development?
Ideally, it would be great to be configurable.